### PR TITLE
Remove usless interface and trait comments

### PR DESCRIPTION
**Pull request by BackEndTea**
Tools like PHPStorm also generate these kinda comments. (And we had a code base filled to the brim with them)

I'd love to add tests for this, but i couldn't get the test suite to work locally.

### CODE REVIEW

File: lib/Doctrine/ruleset.xml [hunk 1]
```diff
@@ -205,6 +205,8 @@
                 <element value="~^(User|Date|Time): \S+\z~i"/>
                 <element value="~^\S+ [gs]etter\.\z~i"/>
                 <element value="~^Class \S+\z~i"/>
```
Comment (ostrolucky) [inline lib/Doctrine/ruleset.xml]:
```suggestion
                 <element value="~^(Class|Interface|Trait) \S+\z~i"/>
```

Comment (BackEndTea) [reply lib/Doctrine/ruleset.xml]: That is probably better, ive consolidated the regexes into one

**Comment by malarzm** [comment]
@BackEndTea as for the tests you can take a look at https://github.com/doctrine/coding-standard/pull/97/files which introduced original ban for `Class` comments and add similar stuff for your changes. Then CI will run the suite for you after committing letting you know if they work or not :)

**Comment by carusogabriel** [comment]
@BackEndTea Just follow the tip that @malarzm gave you, and run `make test` to ensure that the new tests are working 👍

File: tests/php-compatibility.patch [hunk 2]
```diff
@@ -17,8 +17,9 @@ index d1fdd9f..50f5de6 100644
  tests/input/duplicate-assignment-variable.php         1       0
  tests/input/EarlyReturn.php                           6       0
 -tests/input/example-class.php                         34      0
+-tests/input/forbidden-comments.php                    8       0
 +tests/input/example-class.php                         39      0
- tests/input/forbidden-comments.php                    8       0
++tests/input/forbidden-comments.php                    14      0
```
Comment (carusogabriel) [inline tests/php-compatibility.patch]: Added errors:

```sh
 45 | ERROR | [x] Documentation comment contains forbidden comment "Interface Bar".
 47 | ERROR | [ ] Each class must be in a file by itself
 47 | ERROR | [ ] Interface name doesn't match filename; expected "interface forbidden-comments"
 52 | ERROR | [x] Documentation comment contains forbidden comment "Trait Baz".
 54 | ERROR | [ ] Each interface must be in a file by itself
 54 | ERROR | [ ] Trait name doesn't match filename; expected "trait forbidden-comments"
```

**Comment by carusogabriel** [comment]
@lcobucci Mind to help me here with `make update-compatibility-patch`? I can't figure out what I'm doing wrong.

**Comment by carusogabriel** [comment]
@lcobucci Mind to help me here with `make update-compatibility-patch`? I can't figure out what I'm doing wrong.

**Pull request merged**